### ISSUE TITLE
Add Lightgbm Regressor in fish-weight-estimation.ipynb

### ISSUE THREAD

**Issue by Prasadayus** [Niketkumardheeryan/ML-CaPsule#739]
Add Lightgbm Regressor to check if accuracy is increased or not from other model.

**Comment by diaxx1309** [comment]
Hello!! I'd love to work on this issue as part of GSSoC's 2026. I'm currently doing an ML internship and have hands-on experience with Python and scikit-learn. I'll add the lightGBM Regressor to the notebook and compare its accuracy with the existing models. Please assign it to me.
